cms/static/sass/elements/_creative-commons.scss

```diff
@@ -1,10 +1,10 @@
 @font-face {
   font-family: 'CreativeCommons';
-  src: url('../fonts/CreativeCommons/cc.eot');
-  src: url('../fonts/CreativeCommons/cc.eot#iefix') format('embedded-opentype'),
-       url('../fonts/CreativeCommons/cc.woff') format('woff'),
-       url('../fonts/CreativeCommons/cc.ttf') format('truetype'),
-       url('../fonts/CreativeCommons/cc.svg#CreativeCommons') format('svg');
+  src: url($static-path + '/fonts/CreativeCommons/cc.eot');
```

Comment:
This ruleset brings up an interesting case (if I'm understanding things correctly)... 

If an instance owner who is using a theme sets this ``$static-path`` var to something other than the default config, the compiled CSS would be looking for all of these assets starting with that path. If this is the case, an instance owner would have to copy all of these files over to their theme to avoid missing assets, correct?